examples/homography_estimation.py

```diff
@@ -17,9 +20,12 @@
 from torch.utils.data import DataLoader, Dataset
 
 import theseus as th
+from theseus.core.cost_function import AutogradMode, ErrFnType
 from theseus.third_party.easyaug import GeoAugParam, RandomGeoAug, RandomPhotoAug
 from theseus.third_party.utils import grid_sample
 
+warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
Let's add this:

```python
if not sys.warnoptions:
    warnings.filterwarnings("ignore", category=UserWarning)
```

This way if we run as `PYTHONWARNINGS=1 python examples/homography_estimation.py` we will be able to see if there are any warnings that should be addressed (e.g., because of version changes in dependencies, etc.).

Reply:
Added.